packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -760,10 +764,18 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
 
         if (checkTombstone && this.tombstoned) {
             const messageString = `Context is tombstoned! Call site [${callSite}]`;
-            throw new DataCorruptionError(messageString, {
+            const error = new DataCorruptionError(messageString, {
                 errorMessage: messageString,
                 ...safeTelemetryProps,
             });
+
+            this.subLogger.sendErrorEvent({
+                eventName: "Tombstone_DataStore_Changed",
```

Comment:
nit: The usual convention is camel casing - TombstonDatastoreChanged. I would also add the term GC somewhere to make it clear it's GC related which might help OCEs.

Reply:
Actually scratch the part about camel casing. We do have the GC logs with underscore :-).